### PR TITLE
chore: add build targets to makefile and remove travis config

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,3 +8,21 @@ pytest:
 
 pylint:
 	python -m pylint **/*.py
+
+# https://packaging.python.org/tutorials/packaging-projects/
+.PHONY: build
+build:
+	rm dist -r
+	python -m pip install virtualenv
```
Comment (pmalek-sumo) [inline Makefile:16]: Why does this use `python` and below targets use `python3`?

**Comment by pmalek-sumo** [comment]
Can we sidable travis if it's not meant to pass?

File: Makefile (hunk 1 above)
Comment (sumo-drosiek) [reply Makefile:16]: fixed

**Review by pmalek-sumo — APPROVED** [review]
> add build targets to makefile

removes travis config in the same PR :)

**Pull request merged**